src/overlays/actors/ovl_En_Tp/z_en_tp.c

```diff
@@ -58,43 +79,699 @@ static ColliderJntSphInit D_80B22AB4 = {
     1,
     D_80B22A90,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Tp/func_80B20DE0.s")
 
+typedef enum {
+    /* 00 */ TAILPASARAN_DMGEFF_NONE, // No effect
```

Comment:
```suggestion
    /* 00 */ TAILPASARAN_DMGEFF_NONE,
```
The comment probably isn't necessary.